test/index.js

```diff
@@ -28,12 +28,12 @@ const runTest = function(url) {
         error: '>=2000'
       },
       psi: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=2000',
+        error: '>=3500'
       },
       vc85: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=3500',
+        error: '>=5000'
```

Comment:
Ah thanks for pointing it out. fyi, It's time to interactive performance metric, I'm playing around with the numbers because after sometime between friday and today the tti, jumped from 1600s to ~4000.

Trying to figure out where that bump is coming from :/